toolchain/lower/function_context.h

```diff
@@ -194,6 +240,14 @@ class FunctionContext {
   auto CopyObject(SemIR::TypeId type_id, SemIR::InstId source_id,
                   SemIR::InstId dest_id) -> void;
 
+  auto AddGlobalToCurrentFingerprint(llvm::Value* global) {
+    if (function_fingerprint_ && global) {
```

Comment:
Perhaps this and `EmitFinalFingerprint` could use early returns for consistency? It's partly just odd in contrast with `AddCallToCurrentFingerprint`